contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -712,15 +722,15 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         require(token.transfer(destination, amount), "Drain failed");
     }
 
-    function reinitialize(
-        address _NFTContract,
-        address _stakingLogger,
-        address _validatorShareFactory
-    ) external onlyGovernance {
-        NFTContract = StakingNFT(_NFTContract);
-        logger = StakingInfo(_stakingLogger);
-        factory = ValidatorShareFactory(_validatorShareFactory);
-    }
+    // function reinitialize(
+    //     address _NFTContract,
+    //     address _stakingLogger,
+    //     address _validatorShareFactory
+    // ) external onlyGovernance {
+    //     NFTContract = StakingNFT(_NFTContract);
+    //     logger = StakingInfo(_stakingLogger);
+    //     factory = ValidatorShareFactory(_validatorShareFactory);
+    // }
```

Comment:
We should keep this since we may need to change the validator share factory.

Reply:
for validator share factory we have method to update it.

Reply:
keeping this one, removed Factory update function.